src/Evolution/Systems/Cce/Actions/UpdateGauge.hpp

```diff
@@ -21,18 +21,20 @@ namespace Actions {
  * \ingroup ActionsGroup
  * \brief Updates all of the gauge quantities associated with the additional
  * regularity-preserving gauge transformation on the boundaries for a new set of
- * Cauchy coordinates.
+ * Cauchy and partially flat Bondi-like coordinates.
  *
- * \details This action is to be called after `Tags::CauchyCartesianCoords` has
- * been updated, typically via a time step of a set of coordinate evolution
- * equations. It prepares the gauge quantities in the \ref DataBoxGroup for
- * calls to the individual `GaugeAdjustedBoundaryValue` specializations.
+ * \details This action is to be called after `Tags::CauchyCartesianCoords`
+ * and `Tags::InertialCartesianCoords` have been updated, typically via a
+ * time step of a set of coordinate evolution equations. It prepares the
+ * gauge quantities in the \ref DataBoxGroup for calls to the individual
+ * `GaugeAdjustedBoundaryValue` specializations.
  *
  * Internally, this dispatches to `GaugeUpdateAngularFromCartesian`,
  * `GaugeUpdateJacobianFromCoordinates`, `GaugeUpdateInterpolator`, and
  * `GaugeUpdateOmega` to perform the computations. Refer to the documentation
  * for those mutators for mathematical details.
  */
+template <bool EvolveInertialCoordinates>
```

Comment:
Is this template parameter supposed to be the same as what is currently called `EvolutionMetavars::uses_inverse_coordinates` ?  If so, then the two template parameters should have similar names.